auth-web/src/components/auth/UserManagement.vue

```diff
@@ -212,6 +223,7 @@ export default class UserManagement extends Vue {
 
   private async mounted () {
     this.isLoading = false
+    this.pendingApprovals = this.pendingOrgMembers.length
```

Comment:
This looks good @mengdong19 .  I have one suggestion: use a getter for the pendingApprovals length.  That way you can bind to the getter directly, and it will always reflect the current length.  As is, the length will be set when the component mounts, but won't update after that, unless you reload.